docs/developer_guide.md

```diff
@@ -95,7 +107,9 @@ Install latest VS2017 (15.9 or later) from here: http://visualstudio.com/downloa
 
 #### SDK
 
-While WinUI is designed to work against many versions of Windows, you will need a fairly recent SDK in order to build WinUI. It's required that you install the 16299, 17134 and 17763 SDKs.
+While WinUI is designed to work against many versions of Windows, you will need 
+a fairly recent SDK in order to build WinUI. It's required that you install the 
+16299, 17134 and 17763 SDKs. You can download these from here: https://developer.microsoft.com/en-US/windows/downloads/windows-10-sdk
```

Comment:
It's not obvious where to get 16299 and 17134. Can we add a link?

Reply:
I believe if you have the latest you're covered for the older ones. I _think_. At least when I tried on a relatively clean laptop all I had to do was get the latest. Do you know if there are additional steps required?

Reply:
Using the VS installer is preferable. We should instruct people on installing the sdks from there.